File: two_sum/two_sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        ret = []
        for i in range(len(nums)):
            second = target - nums[i]
            for j in range(i+1, len(nums)):
                if(second == nums[j]):
                    ret.append(i)
                    ret.append(j) 
                    return ret

File: two_sum/two_sum.cpp
class Solution {
	public:
		vector<int> twoSum(vector<int>& nums, int target) {
			vector<int> a; // array of the indices of two numbers such that they add up to target
			bool found = true;
			for(int i = 0; i < nums.size() && found; i++) {
				int second = target - nums[i];
				for (int j = i+1; j < nums.size(); j++) {
					if(second == nums[j]) {
						a.push_back(i);
						a.push_back(j);
						found = 0;
						break;
					}
				}
			}
			return a;
		}
};
